tests/unit/setup-unit-test.ts

```diff
@@ -74,7 +75,12 @@ module.exports = async () => {
     path.join(vcastEnvPath, "unit.cpp")
   );
 
-  const clicastTemplateCommand = `cd ${vcastEnvPath} && ${clicastExecutablePath.trimEnd()} -l C template GNU_CPP11_X`;
+  const clicastTemplateCommand = `cd ${vcastEnvPath} && ${clicastExecutablePath.trimEnd()} -l C template GNU_CPP11_X option VCAST_CODED_TESTS_SUPPORT TRUE`;
+
+  await promisifiedExec(
+    `cd ${vcastEnvPath} &&  ${clicastExecutablePath.trimEnd()} -lc option VCAST_CODED_TESTS_SUPPORT TRUE`
+  );
```

Comment:
We likely only want to do this when we're vc24

Reply:
Also, we should make this work like the other calls (i.e., log the stderr if it exists).

Reply:
Reworked in 770736c06264d30fd8e6d29f6695849bfad10960 .